src/hammer-vlsi/hammer_vlsi/cli_driver.py

```diff
@@ -856,6 +859,21 @@ def auto_action(driver: HammerDriver, append_error_func: Callable[[str], None])
 
         return driver, errors
 
+    @staticmethod
+    def generate_build_inputs(driver: HammerDriver, append_error_func: Callable[[str], None]) -> Optional[dict]:
+        """
+        Generate the build tool artifacts for this flow, specified by the "vlsi.core.build_system" key.
+
+        :param driver: The HammerDriver object which has parsed the configs specified by -p
+        :param append_error_func: The function to use to append an error (unused).
+        :return: A build-system-specific dictionary
```

Comment:
Currently doesn't it return the dependency graph?

Reply:
Yes, but it doesn't have to. I'll go ahead and change it- I doubt we'll ever return anything esle.